.github/workflows/tests/BUILD

```diff
@@ -0,0 +1,24 @@
+# COMMENT
+
+shell_sources(name="tests")
+
+def _run_test(filename):
+    filepath = build_file_dir() / 'smoke_test_auth_cherry_picker.sh'
+    return f"chmod +x {filepath} && {filepath}"
+
+
+
+experimental_test_shell_command(
```

Comment:
Did you consider using https://www.pantsbuild.org/docs/reference-shunit2_tests here? I think this might simplify the behaviour when adding more tests: just add another `..._test.sh` file (and any requisite dependency overrides), and it'll do the right thing, rather than needing to duplicate this content. E.g.

```python
shunit2_tests(
    dependencies=["//build-support/bin/act:act-at-root"],
    overrides={
       "auto_cherry_picker_test.sh": dict(dependencies=[".github/workflows/auto-cherry-picker.yaml"]),
    },
)
```

Reply:
I decided last night we should just use Python tests. Subprocessing is annoying, but the assertions and other stuff gets easier